packages/components/src/Tooltip/Tooltip.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { generateSnapshots } from "@chanzuckerberg/story-utils";
+import { userEvent } from "@storybook/testing-library";
+import { composeStories } from "@storybook/testing-react";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import * as TooltipStoryFile from "./Tooltip.stories";
+
+const { Interactive } = composeStories(TooltipStoryFile);
+
+describe("<Tooltip />", () => {
+  generateSnapshots(TooltipStoryFile, {
+    getElement: (wrapper) => {
+      return wrapper.baseElement;
```

Comment:
May want to leave a comment that Tippy renders the tooltip as a child of `<body>`, so that people don't unnecessarily copy and paste this.

Reply:
Added comment, thanks